artiq/examples/kasli_basic/repository/kasli_tester.py

```diff
@@ -16,11 +18,18 @@ def chunker(seq, size):
 
 
 def is_enter_pressed() -> TBool:
-    if select.select([sys.stdin,], [], [], 0.0)[0]:
-        sys.stdin.read(1)
-        return True
-    else:
-        return False
+    if sys.platform == 'linux2':
+        if select.select([sys.stdin,], [], [], 0.0)[0]:
+            sys.stdin.read(1)
+            return True
+        else:
+            return False        
```

Comment:
This code can be factored:
``
if os.name == "nt":
   is_key_pressed = lambda: select.select([sys.stdin,], [], [], 0.0)[0]
else:
  import msvcrt
  is_key_pressed = msvcrt.kbhit
```